backend/api/Startup.cs

```diff
@@ -138,6 +141,20 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IKeycloakRequestClient, KeycloakRequestClient>();
 
             services.AddAutoMapper(typeof(Startup));
+
+            services.AddHealthChecks()
+                .AddCheck("liveliness", () => HealthCheckResult.Healthy())
+                .AddSqlServer(builder.ConnectionString, tags: new[] { "services" });
+            //TODO: Add a health check for keycloak connectivity.
+            services.AddHealthChecksUI(setupSettings: setup =>
```

Comment:
This adds an interface that provides the following functionality:
1) View the status of health checks within the application.
2) Send notifications in the event that one or more health checks fail, see below for more details.

Reply:
Our openshift pipeline should notify us if the health checks fail during the deployment process, but this provides the extra functionality of notifying us if the health checks fail during normal execution.

Reply:
I'm very curious to see all this working.

Reply:
I also like that we get a nice UI but I'd suggest if you could check it out a bit so it can't get abused to spam our server(s)